### PR TITLE
Converting all fedorapeople repos and adding new config setting

### PR DESCRIPTION
**Pull request by Griffin-Sullivan**
This PR includes the changes for all of the fedorapeople repos except FAKE_5_YUM_REPO, FAKE_7_PUPPET_REPO, and REPO_DISCOVERY_URL. I've also added a new setting under robottelo in the robottelo.properties file for fixtures_url.

======================================= test session starts ========================================
platform linux -- Python 3.8.3, pytest-4.6.9, py-1.8.0, pluggy-0.13.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/gsulliva/Programming/robottelo
plugins: mock-1.10.4, services-1.3.1, forked-1.1.3, xdist-1.32.0, cov-2.9.0
collecting ... 2020-08-11 15:23:39 - conftest - DEBUG - Collected 22 test cases
collected 22 items                                                                                 

cli/test_contentview.py .                                                                    [  4%]
api/test_contentviewfilter.py .                                                              [  9%]
cli/test_activationkey.py .                                                                  [ 13%]
api/test_contentviewversion.py .                                                             [ 18%]
api/test_repository.py .                                                                     [ 22%]
api/test_errata.py .                                                                         [ 27%]
cli/test_repository.py .                                                                     [ 31%]
api/test_repository.py .                                                                     [ 36%]
ui/test_contenthost.py .                                                                     [ 40%]
api/test_contentmanagement.py .                                                              [ 45%]
ui/test_contentview.py ..                                                                    [ 54%]
cli/test_repository.py .                                                                     [ 59%]
api/test_contentmanagement.py .                                                              [ 63%]
api/test_repository.py .                                                                     [ 68%]
cli/test_repository.py ..                                                                    [ 77%]
api/test_repository.py .                                                                     [ 81%]
ui/test_repository.py ..                                                                     [ 90%]
cli/test_contentview.py .                                                                    [ 95%]
cli/test_repository.py .                                                                     [100%]

========================================= warnings summary =========================================
tests/foreman/ui/test_contenthost.py::test_positive_remove_package
tests/foreman/ui/test_contentview.py::test_positive_add_all_security_errata_by_id_filter
tests/foreman/ui/test_contentview.py::test_positive_add_all_security_errata_by_id_filter
tests/foreman/ui/test_contentview.py::test_positive_depsolve_with_module_errata
tests/foreman/ui/test_contentview.py::test_positive_depsolve_with_module_errata
tests/foreman/ui/test_contentview.py::test_positive_depsolve_with_module_errata
tests/foreman/ui/test_contentview.py::test_positive_depsolve_with_module_errata
tests/foreman/ui/test_contentview.py::test_positive_depsolve_with_module_errata
tests/foreman/ui/test_repository.py::test_positive_discover_repo_via_existing_product
tests/foreman/ui/test_repository.py::test_positive_discover_repo_via_existing_product
tests/foreman/ui/test_repository.py::test_positive_discover_repo_via_existing_product
tests/foreman/ui/test_repository.py::test_positive_discover_repo_via_existing_product
  /home/gsulliva/.local/lib/python3.8/site-packages/widgetastic/widget/select.py:131: DeprecationWarning: The unescape method is deprecated and will be removed in 3.5, use html.unescape() instead.
    normalize_space(parser.unescape(option))

tests/foreman/ui/test_contentview.py::test_positive_add_all_security_errata_by_id_filter
tests/foreman/ui/test_contentview.py::test_positive_add_all_security_errata_by_id_filter
tests/foreman/ui/test_contentview.py::test_positive_depsolve_with_module_errata
tests/foreman/ui/test_contentview.py::test_positive_depsolve_with_module_errata
tests/foreman/ui/test_contentview.py::test_positive_depsolve_with_module_errata
tests/foreman/ui/test_contentview.py::test_positive_depsolve_with_module_errata
tests/foreman/ui/test_contentview.py::test_positive_depsolve_with_module_errata
tests/foreman/ui/test_contentview.py::test_positive_depsolve_with_module_errata
tests/foreman/ui/test_contentview.py::test_positive_depsolve_with_module_errata
  /home/gsulliva/.local/lib/python3.8/site-packages/widgetastic/widget/base.py:829: DeprecationWarning: inspect.getargspec() is deprecated since Python 3.0, use inspect.signature() or inspect.getfullargspec()
    c_args, c_varargs, c_keywords, c_defaults = inspect.getargspec(condition)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
============================ 22 passed, 21 warnings in 3620.78 seconds =============================

### CODE REVIEW

File: robottelo/constants.py [hunk 1]
```diff
@@ -2,6 +2,13 @@
 """Defines various constants"""
 from nailgun import entities
 
+from robottelo.config import settings
+
+if not settings.configured:
+    settings.configure()
+
+FIXTURES_URL = settings.fixtures_url
```
Comment (pondrejk) [inline robottelo/constants.py:10]: fixtures_url sounds a bit vague, I'd change the name to make it clearer what it represents, something like repo_hosting_url?

**Comment by pondrejk** [comment]
@Griffin-Sullivan some notes:
- Please add the new setting to robottelo.properties.sample, with some description ideally

 Otherwise no problem with the changes, but some coordination is needed:
- I assume we have a working url for hosting. We first need to add it as a setting to robottelo.properties used in automation (gitlab), if we merge this PR earlier we will break many tests :)
- also please communicate this change to the whole team (mailing list I guess) so that everyone can update their local properties file 

Cheers

**Comment by mshriver** [comment]
Thanks @Griffin-Sullivan for your great contribution and moving us so far forward on this effort!

As you no longer have internal access, I'll be "taking over" this PR to some degree, adding commits concerning any comments or required changes to get this merged.  

If need be I will open a new PR - but will be sure to maintain your commits!

File: robottelo/constants.py (hunk 1 above)
Comment (jyejare) [reply robottelo/constants.py:10]: Changed to `repos_hosting_url`, hope that makes sense!

**Comment by jyejare** [comment]
@Griffin-Sullivan Taking over your PR and creating a new PR [1]. But no worries you are still an author :) 

[1] : https://github.com/SatelliteQE/robottelo/pull/7993

Closing this PR in favor of new.

**Pull request closed without merging**